kernel/rros/factory.rs

```diff
@@ -347,23 +350,21 @@ fn create_element_device(
     };
 
     let _res = do_element_visibility(e.clone(), fac, &mut rdev);
-    let e_clone = e.clone();
-    let mut e_mut = e_clone.borrow_mut();
-    let filp = e_mut.fpriv.filp.as_mut().unwrap().get_ptr();
-    pr_debug!("the address of filp location 7 is {:p}, {:p}", filp, &filp);
-
-    // if (!rros_element_is_public(e) && !rros_element_has_coredev(e)) {
-    // if rros_element_is_public(e.clone()) == false && rros_element_has_coredev(e.clone()) == false {
-    // unsafe { bindings::fd_install(e_mut.fpriv.efd.reserved_fd(), filp) };
-    // e.fpriv.efd.commit(File{ptr: filp});
-    // }
-    fd_install(e_mut.fpriv.efd.reserved_fd(), filp);
-    pr_debug!("the address of filp location 8 is {:p}, {:p}", filp, &filp);
-    // 	e->refs++;
-    // 	fd_install(e->fpriv.efd, e->fpriv.filp);
-    // }
-    // let e_clone = e.clone();
-    // let mut e_mut = e_clone.borrow_mut();
+    if !rros_element_is_public(e.clone()) && !rros_element_has_coredev(e.clone()) {
+        let e_clone = e.clone();
+        let mut e_mut = e_clone.borrow_mut();
+        e_mut.refs += 1;
```

Comment:
We use `Rc` here to record the reference count of e, so the `e_mut.refs` may not be correct.

Reply:
Thanks for reviewing! `e_mut` is a `RrosElement` here, and `refs` is a field in `RrosElement`. I will use this field in `clock.rs`, changing this field here is necessary. This code will not influence any other code except `clock.rs`.